Boa.Constrictor.Selenium/Tasks/SwitchFrame.cs

```diff
@@ -80,24 +111,25 @@ public override void PerformAs(IActor actor, IWebDriver driver)
         /// <param name="obj">The other object.</param>
         public override bool Equals(object obj) =>
             obj is SwitchFrame frame &&
-            Locator.Equals(frame.Locator) &&
+            Locators.Equals(frame.Locators) &&
```

Comment:
`Equals` on a list will check references, not values.

https://stackoverflow.com/questions/22173762/check-if-two-lists-are-equal